bowjson.go

```diff
@@ -94,7 +95,13 @@ func (b *bow) NewValuesFromJSON(jsonB JSONBow) error {
 	*/
 
 	for fieldIndex, field := range jsonB.Schema.Fields {
-		if _, ok := mapArrowNameToBowTypes[field.Type]; ok {
+		ok := false
```

Comment:
why switch from a map key search to a map value search, did `mapArrowNameToBowTypes` disappear ?

Reply:
My mistake, I created back `mapArrowNameToBowTypes`.